draft-ietf-quic-recovery.md

```diff
@@ -1243,8 +1245,18 @@ OnLossDetectionTimeout():
     crypto_count++
   else:
     // PTO
-    SendOneOrTwoPackets()
     pto_count++
+    if (any ack-eliciting frames available to send):
+      SendOneOrTwoPackets()
+    else:
+      // Mark all packets in flight as lost
```

Comment:
This pseudocode makes an excellent argument that we should recommend sending a PING to establish loss.

Reply:
Why? Because you'd have to mark packets as lost?